src/services/usersService.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { UserModel } = require('../models/userModel.js');
+const { sequelize } = require('sequelize');
+
+const getAll = async() => {
+  const users = await UserModel.findAll({
+    order: [
+      'spentAt',
+    ],
+  });
+
+  return users;
+};
+
+const getById = async(userId) => {
+  const foundUser = await UserModel.findByPk(userId);
+
+  return foundUser;
+};
+
+const addUser = async(name) => {
+  const createdUser = await UserModel.create({ name });
+
+  return createdUser;
+};
+
+const removeUser = async(userId) => {
+  if (!getById(userId)) {
+    return false;
+  }
+
+  await UserModel.destroy({
+    where: {
+      id: userId,
+    },
+  });
+
+  return true;
+};
+
+const updateUser = async(userId, name) => {
+  if (!getById(userId)) {
+    return null;
+  }
+
+  const updatedUser = await UserModel.update({ name }, {
+    where: {
+      id: userId,
+    },
+  });
```

Comment:
Model.update doesn't return the row instance. Check the [doc](https://sequelize.org/api/v6/class/src/model.js~model#static-method-update).

What you need is to take the instance from getById, and call [update](https://sequelize.org/api/v6/class/src/model.js~model#instance-method-update) on it, not the static model method